pom.xml

```diff
@@ -1981,6 +1990,21 @@
           </configuration>
         </plugin>
 
+        <plugin>
+          <groupId>com.hubspot.maven.plugins</groupId>
+          <artifactId>prettier-maven-plugin</artifactId>
+          <version>0.6-SNAPSHOT</version>
```

Comment:
We'll probably want to cut a release soon so basepom isn't depending on snapshots